ALCF/helpers.sh

```diff
@@ -122,6 +122,19 @@ setup_run_cmd() {
     if [[ "${SP}" -ge 2 ]]; then
         export DEFAULTS="${DEFAULTS} --ds-sequence-parallel-size ${SP} --force-ds-sequence-parallel"
     fi
+
+    # [hacky]: to disable Llama-type architectures, toggle via:
+    # `NO_LLAMA=1 bash train_llama_alcf.sh`
+    if [[ -z "${NO_LLAMA:-}" ]]; then
+        llama_flags="${LLAMA_ARGS}\
+            --num-key-value-heads ${NUM_KV_HEAD} \
+            --ffn-hidden-size ${FFN_HIDDEN_SIZE} \
+            "
+    else
+        echo "!! Running in NO_LLAMA MODE !!"
+        llama_flags=""
+    fi
```

Comment:
basically, at the very end (once all of the necessary variables have been defined / instantiated), we:

- check if `"${NO_LLAMA:-}"` is **empty** in current environment (on 128), and if so, set:

    ```bash
    llama_flags="${LLAMA_ARGS} --num-key-value-heads ${NUM_KV_HEAD} --ffn-hidden-size ${FFN_HIDDEN_SIZE}"
    ```

- otherwise, set:
    
    ```bash
    llama_flags=""
    ```

and then pass these "${llama_flags}" directly to [`pretrain_gpt_alcf.py`](https://github.com/argonne-lcf/Megatron-DeepSpeed/blob/main/pretrain_gpt_alcf.py)